TLM/TLM/Manager/Impl/LaneArrowManager.cs

```diff
@@ -182,11 +182,11 @@ public class LaneArrowManager
         private static void RecalculateFlags(uint laneId) {
             NetLane[] laneBuffer = NetManager.instance.m_lanes.m_buffer;
             ushort segmentId = laneBuffer[laneId].m_segment;
-            NetAI ai = GetSeg(segmentId).Info.m_netAI;
+            NetAI ai = segmentId.ToSegment().Info.m_netAI;
 #if DEBUGFLAGS
             Log._Debug($"Flags.RecalculateFlags: Recalculateing lane arrows of segment {segmentId}.");
 #endif
-            ai.UpdateLanes(segmentId, ref GetSeg(segmentId), true);
+            ai.UpdateLanes(segmentId, ref segmentId.ToSegment(), true);
```

Comment:
no need to do it twice for the same segment.

Reply:
this is not performance critical. in any case I am not doing anything the old code didn't do.

Reply:
yeah I would have probably done it in a separate commit too